Sources/Implementation/DefaultDecisionService.swift

```diff
@@ -80,12 +85,19 @@ class DefaultDecisionService: OPTDecisionService {
         return bucketedVariation
     }
     
-    func isInExperiment(config: ProjectConfig, experiment: Experiment, userId: String, attributes: OptimizelyAttributes) -> Bool {
+    func isInExperiment(config: ProjectConfig, experiment: Experiment, userId: String, attributes: OptimizelyAttributes, logType: evaluationLogType = .experiment, loggingKey: String? = nil) -> Bool {
         
         var result = true   // success as default (no condition, etc)
+        let evType = logType.rawValue
+        
+        var finalLoggingKey = experiment.key
+        if let key = loggingKey {
+            finalLoggingKey = key
+        }
         
         do {
             if let conditions = experiment.audienceConditions {
+                logger.d(.evaluatingAudiencesCombined(evType, finalLoggingKey, Utils.getConditionString(conditions: conditions)))
```

Comment:
```suggestion
                logger.d {
                   return .evaluatingAudiencesCombined(evType, finalLoggingKey, Utils.getConditionString(conditions: conditions)))
                }
```

"getConditionsString" will be expensive. Let's avoid this for normal flow (logLevel < .debug).
https://github.com/optimizely/swift-sdk/blob/bc787345a2932833d88ac9881ec6fa2f3bcdc231/Sources/Customization/Protocols/OPTLogger.swift#L51